specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/managedInstances.json

```diff
@@ -351,6 +351,26 @@
           "format": "int32",
           "description": "The maximum storage size in GB.",
           "type": "integer"
+        },
+        "collation": {
+          "description": "Collation of the managed instance.",
+          "type": "string",
+          "x-ms-mutability": [
```

Comment:
I thought this was made readonly now

Reply:
Yes, the Collation property is made read-only a three weeks ago. Mike, which sync point you've used for generating the swagger changes?